index.njk

```diff
@@ -14,4 +14,11 @@ menu:
 
 <hr>
 
+<h2>Pinned Post:</h1>
+
+{% set pinnedpost = search.pages("'pinned'") %}
```

Comment:
You're using the `pinned` tag to select the pages. This is a different thing to tags, so I'd use a different identifier. For example `pinned=true`.

Reply:
Yes, but how to do it

Reply:
Just add the `pinned: true` value in the frontmatter of the posts, and select it with `search.pages("pinned=true")`